ajax/libs/hprose-html5/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "hprose-html5",
+  "filename": "hprose-html5.js",
+  "version": "1.5.3",
+  "description": "Hprose is a High Performance Remote Object Service Engine. It is a modern, lightweight, cross-language, cross-platform, object-oriented, high performance, remote dynamic communication middleware. It is not only easy to use, but powerful. You just need a little time to learn, then you can use it to easily construct cross language cross platform distributed application system.",
+  "homepage": "https://github.com/andot/hprose",
+  "keywords": [
+    "hprose",
+    "rpc",
+    "webservice",
+    "websocket",
+    "http",
+    "ajax",
+    "json",
+    "jsonrpc",
+    "cross-language",
+    "cross-platform",
+    "cross-domain",
+    "html5",
+    "serialize",
+    "serialization",
+    "protocol",
+    "web",
+    "service",
+    "framework",
+    "library",
+    "game",
+    "communication",
+    "middleware"
+  ],
+  "maintainers": [
+    {
+      "name": "Ma Bingyao",
+      "email": "andot@hprose.com",
+      "web": "http://hprose.com/"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/hprose/hprose-html5.git"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/hprose/hprose-html5.git",
+    "basePath": "dist/",
+    "files": [
+      "hprose-html5.js"
+    ]
+  },
+  "npmName": "hprose-html5",
```

Comment:
OK, so actually the importer should be even smarter: to check if the `autoupdate` field is there or not. If it is, then it will **not** add the `npmFileMap` in the `package.json` file, right?